packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -18,6 +18,7 @@ import 'viewport_offset.dart';
 
 const double _kCaretGap = 1.0; // pixels
 const double _kCaretHeightOffset = 2.0; // pixels
+const double _kIOSCursorRadius = 1.0; // pixels
```

Comment:
Try to keep platform specific aesthetic things out of the rendering and widget layers as much as possible. Have this passed in from TextField and CupertinoTextField. There are construction args on the editable.